app/controllers/posts_controller.rb

```diff
@@ -1,11 +1,11 @@
 class PostsController < ApplicationController
   def index
-    @post = User.find_by(id: params['user_id'])
+    @post = User.includes(:posts).find_by(id: params['user_id'])
```

Comment:
- [ ] @danielsafari143 and @BryteLitty Great work trying to fix the N + 1 problem, however, you did it wrongly, to fix the issue we are supposed to fetch all user's posts and comments at once ge using a code like `@posts = @user.posts.includes(:comments)` k.kindly consider fixing the issue and update the screenshots as well.